### ISSUE TITLE
Slow popup

### ISSUE THREAD

**Issue by bitcoiners** [GetScatter/ScatterWebExtension#109]
Tried using scatter with several websites. It is seems there is something slow about current functionality relating to scatter popup window taking a long time before opening, sometimes over 10 seconds.
Same issue on monstereos.io and decentwitter.com

**Comment by nsjames** [comment]
Yeah, some nodes are slow and eosjs has to go grab the ABI from the site ( which for some reason comes back as a whopping 1+ mb JSON ). I've been meaning to add a ticket on eosjs about it.

**Comment by nsjames** [comment]
https://github.com/EOSIO/eos/issues/4222
